doc/content/devices/models/mkr1310/temp.md

```diff
@@ -1,104 +1,42 @@
+- Breadboard
 
-Once you have the components, wire it up like so:
+Once you have the components, wire it up like shown on the image on the right:
 - **Arduino 5V** -> **Plus DHT11**
 - **Arduino GND** -> **Minus DHT11**
 - **Arduino Pin 6** -> **Signal DHT11**
 
-Illustration on the right displays the wiring as well.
-
 {{< note "Double check your own sensor's pinout, it can vary between modules. If you are using a bare DHT11, you might need to use a resistor." />}}
 
-## Creating the code
+## Programming the MKR 1310
 
-Before we continue, you need to make sure you have the **DHT sensor library by Adafruit** installed: 
+Before we continue, you need to make sure you have the **DHT sensor library by Adafruit** installed in the Arduino IDE:  
 
 {{< figure src="../dht-lib.png" alt="DHT sensor library by Adafruit">}}
 
-Next let's go through the code step by step and explain what all the fields mean. The completed code can be found below it.
-
-### The code explained
-
-First we initialize the **MKRWAN** and **DHT** libraries we need for this code. After that is the creation of the LoRa modem. Then we set up the variables for the `AppEUI` and `AppKey`. **Note that you need to add the AppEUI and AppKey from {{% tts %}} between the double quotes**. Finally we setup the DHT component with `6` as the pin and `DHT11` as the sensor type.
-
-```cpp
-#include <MKRWAN.h>
-#include <DHT.h>
-
-LoRaModem modem(Serial1);
-
-String appEui = "";
-String appKey = "";
-
-DHT dht(6, DHT11);
-```
-
-In the `void setup ()` we initialize the Serial communication first. Then we set up the modem to use the correct regional band (**Note that you need to change this to whichever region you're in**). Using `dht.begin()` we initialize the dht component. Finally using `modem.joinOTAA` the device will join the LoRaWAN network.
-
-```cpp
-void setup() {
-  // put your setup code here, to run once:
-  Serial.begin(115200);
-  while (!Serial);
-  // change this to your regional band (eg. US915, AS923, ...)
-  if (!modem.begin(EU868)) {
-    Serial.println("Failed to start module");
-    while (1) {}
-  };
-  
-  dht.begin();
-
-  int connected = modem.joinOTAA(appEui, appKey);
-  if (!connected) {
-    Serial.println("Something went wrong; are you indoor? Move near a window and retry");
-    while (1) {}
-  }
-}
-```
-
-In the `void loop()` we initialize the variables containing the values of Temperature and Humidity. We multiply these values by 100, since we can not send decimal numbers directly over LoRaWAN. Using the `byte payload` function we contain all the values we need for sending. Then using `mode.write` we finally send the package off to the network, ready to be received by {{% tts %}}.
+Next is the code itself. You can find it below, but you do need to change a few things. 
 
-```cpp
-void loop() {
-  int t = dht.readTemperature() * 100;
-  int h = dht.readHumidity() * 100;
+First you have to **add the AppEUI and AppKey from {{% tts %}} between the double quotes**. 
```

Comment:
I actually meant in-code comments, such as:

`String appEui = "123456789ABCDEF";  #Fill in with your AppEUI`

Something like that. Same thing for regional band part

Reply:
I felt like users would not notice that but sure ill just say "check the in-code comments"